index.md

```diff
@@ -1,4 +1,150 @@
 ---
 layout: docs
-title:  "Documentation Title"
----
\ No newline at end of file
+title:  " Content Strategy for the RAC’s Research & Engagement Program"
+---
+
+To be reviewed annually.
+Last approved 11/23.
+
+# Research & Engagement Program Vision
```

Comment:
You'll want to change your heading 1s (#) to heading 2s (##), because the t[itle is the heading 1 on the page](https://docs.rockarch.org/docs-guide/add-content#page-title-and-layout)